_posts/2025-01-15-moving-data-between-JBOD-disks-using-cruise-control.md

```diff
@@ -0,0 +1,546 @@
+---
+layout: post
+title: "Moving data between JBOD disks using Cruise Control"
+date: 2025-01-15
+author: shubham_rawat
+---
+
+Apache Kafka is a platform which provides durability and fault tolerance by storing messages on disks and JBOD storage is one of the storage configuration types supported by Strimzi.
+The JBOD data storage configuration allows Kafka brokers to make use of multiple disks.
+Using JBOD storage, you can increase the data storage capacity for Kafka nodes, which can further lead to performance improvements.
+If you plan to remove a disk that contains partition replicas, the data must be safely moved to other disks first.
```

Comment:
You talk about _Moving data between JBOD disks_. So it would be good to include also adding volumes  -even if briefly in one or two sentences and link to the docs.

```suggestion
It might happen that you need to add or remove volumes to increase or shrink the overall capacity and performance of the Kafka cluster.
When adding new volumes, you first need to add the volume and then move some of the data to to.
That can be done using the _intrabroker_ rebalance: TOTO Link to docs.
When removing volumes, you have to first safely move the data to other volumes first.
```